Portal/Views/Waste/DuringWhichMonth.cshtml

```diff
@@ -0,0 +1,61 @@
+﻿@using Portal.Models;
+@using Portal.Resources;
+@model Portal.Models.DuringWhichMonthRequestViewModel
+
+@{
+    ViewData["Title"] = @WhichQuarterResources.PageTitle;
+}
+
+<title>@ViewData["Title"] @WhichQuarterResources.PageTitle</title>
+
+<a href="/" class="govuk-back-link">Back</a>
+<form method="post">
+    <input type="hidden" asp-for="JourneyId" />
+    <div class="govuk-error-summary @(ViewContext.ViewData.ModelState.IsValid ? "govuk-visually-hidden" : string.Empty)" data-module="govuk-error-summary">
+        <div role="alert">
+            <h2 class="govuk-error-summary__title">
+                There is a problem
+            </h2>
+            <div class="govuk-error-summary__body">
+                <ul class="govuk-list govuk-error-summary__list">
+                    <li>
+                        <a href="#@Model.Quarter.First().Key">@WhichQuarterResources.ErrorMessage</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </div>
+    <div class="govuk-form-group @(!ViewContext.ViewData.ModelState.IsValid ? "govuk-form-group--error" : string.Empty)">
+        <fieldset class="govuk-fieldset" aria-describedby="month-hint">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                <span class="govuk-caption-l">
+                    @WhichQuarterResources.Paper
+                </span>
+                <h1 class="govuk-fieldset__heading">
+                    @WhichQuarterResources.Heading
+                </h1>
+            </legend>
+            <div id="month-hint" class="govuk-hint">
+                @WhichQuarterResources.HintText
+            </div>
+            <div class="govuk-radios" data-module="govuk-radios">
+                <p class="govuk-error-message">
+                    <span asp-validation-for="SelectedMonth"></span>
+                </p>
+                @foreach (var element in Model.Quarter)
+                {
+                    <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="@element.Key" name="SelectedMonth" type="radio" value="@element.Key" aria-describedby="@element.Key">
+                        <label class="govuk-label govuk-radios__label" for="@element.Key">
+                            @element.Value
+                        </label>
+                    </div>
+                }
+            </div>
+        </fieldset>
+    </div>
```

Comment:
Would be good if we can use the radios tag in the long run, but again, let's not worry about it too much at the moment